data/account/participation.go

```diff
@@ -203,19 +202,24 @@ func (part PersistedParticipation) Persist() error {
 	voting := part.Voting.Snapshot()
 	rawVoting := protocol.Encode(&voting)
 
-	return part.Store.Atomic(func(ctx context.Context, tx *sql.Tx) error {
+	err := part.Store.Atomic(func(ctx context.Context, tx *sql.Tx) error {
 		err := partInstallDatabase(tx)
 		if err != nil {
-			return fmt.Errorf("Participation.persist: failed to install database: %v", err)
+			return fmt.Errorf("failed to install database: %w", err)
```

Comment:
this is nice

Reply:
I'm trying to figure if this was a sarcastic comment or not 😅

Reply:
No, it's not. :)  I've recently encountered a situation where `%w` would have been useful.